boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -0,0 +1,237 @@
+   :app: samples/hello_world
+   :board: ttgo_t8c3
+   :goals: build
+   :west-args: --sysbuild
+   :compact:
+
+By default, the ESP32 sysbuild creates bootloader (MCUboot) and application
+images. But it can be configured to create other kind of images.
+
+Build directory structure created by sysbuild is different from traditional
+Zephyr build. Output is structured by the domain subdirectories:
+
+.. code-block::
+
+  build/
+  ├── hello_world
+  │   └── zephyr
+  │       ├── zephyr.elf
+  │       └── zephyr.bin
+  ├── mcuboot
+  │    └── zephyr
+  │       ├── zephyr.elf
+  │       └── zephyr.bin
+  └── domains.yaml
+
+.. note::
+
+   With ``--sysbuild`` option the bootloader will be re-build and re-flash
+   every time the pristine build is used.
+
+For more information about the system build please read the :ref:`sysbuild` documentation.
+
+Manual build
+============
+
+During the development cycle, it is intended to build & flash as quickly possible.
+For that reason, images can be build one at a time using traditional build.
+
+The instructions following are relevant for both manual build and sysbuild.
+The only difference is the structure of the build directory.
+
+.. note::
+
+   Remember that bootloader (MCUboot) needs to be flash at least once.
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: ttgo_t8c3
+   :goals: build
+
+The usual ``flash`` target will work with the ``ttgo_t8c3`` board
+configuration. Here is an example for the :ref:`hello_world`
+application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: ttgo_t8c3
+   :goals: flash
+
+The default baud rate for the Lilygo ttgo t8-c3 is set to 1500000bps. If experiencing issues when flashing,
+try using different values by using ``--esp-baud-rate <BAUD>`` option during
+``west flash`` (e.g. ``west flash --esp-baud-rate 115200``).
+
+You can also open the serial monitor using the following command:
+
+.. code-block:: shell
+
+   west espressif monitor
+
+After the board has automatically reset and booted, you should see the following
+message in the monitor:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
+   Hello World! ttgo_t8c3
+
+Sample applications
+===================
+
+The following samples will run out of the box on the ttgo t8-c3 board.
+
+To build the blinky sample:
+
+.. zephyr-app-commands::
+   :tool: west
+   :app: samples/basic/blinky
+   :board: ttgo_t8c3
+   :goals: build
+
+To build the bluetooth beacon sample:
+
+.. zephyr-app-commands::
+   :tool: west
+   :app: samples/bluetooth/beacon
+   :board: ttgo_t8c3
+   :goals: build
+
+
+Related Documents
+*****************
+.. _`Lilygo TTGO T8-C3 schematic`: https://github.com/Xinyuan-LilyGO/T8-C3/blob/main/Schematic/T8-C3_V1.1.pdf
+.. _`Lilygo github repo <https://github.com/Xinyuan-LilyGo>`
```

Comment:
PLease, modify this line to:  
.. _`Lilygo github repo` https://github.com/Xinyuan-LilyGo

Reply:
Hi Sylvio,
I updated the link (removed the brackets).